reference/shipping_provider.yml

```diff
@@ -27,17 +27,23 @@ info:
     - [Shipping Methods](/api-reference/store-management/shipping-api/shipping-method)
     - [Shipping Carriers](/api-reference/store-management/shipping-api/shipping-carrier)
   termsOfService: ''
-host: api.bigcommerce.com
+host: example.com
```

Comment:
@bc-andreadao can we make this an empty string and add a path parameter so that folks can put their app server host URL in?

cc @markcmurphy 

[slack discussion on this topic](https://bigcommerce.slack.com/archives/C57SQ401K/p1655424501455509)